common.blocks/i-bem/__collection/i-bem__collection.js

```diff
@@ -164,7 +164,7 @@ var BemCollection = inherit(/** @lends BemCollection.prototype */{
     /**
      * Returns a new collection comprised of collection on which it is called joined with
      * the collection(s) and/or array(s) and/or entity(es) provided as arguments.
-     * @param {Collection|Array|BemEntity} [args...]
+     * @param {?...(Collection|Array|BemEntity)} args
```

Comment:
What I tried to explain is that use cases below are valid according to `{?Collection}` notation but they won't work.

~~~js
collection.concat(null).findChildBlock(···); // will throw
collection.concat(undefined).findChildBlock(···); // will throw
~~~

At the same time, this case is only valid for `{Collection} [arg]` and it will work

~~~js
collection.concat().findChildBlock(···); // will work
~~~